### ISSUE TITLE
Rails 5 No Route Matches Error

### ISSUE THREAD

**Issue by beydogan** [sferik/rails_admin#2662]
I just upgraded my rails version to 5.0.0.rc2 from 4.2.6 and getting the following error when I navigated to rails admin page;

```ruby
No route matches {:action=>"index", :controller=>"rails_admin/main", :id=>nil, :model_name=>"user"}
```

But I can see routes when I run `rake routes`

```
> rake routes
..
Routes for RailsAdmin::Engine:
  dashboard GET         /                                      rails_admin/main#dashboard
      index GET|POST    /:model_name(.:format)                 rails_admin/main#index
        new GET|POST    /:model_name/new(.:format)             rails_admin/main#new
     export GET|POST    /:model_name/export(.:format)          rails_admin/main#export
bulk_delete POST|DELETE /:model_name/bulk_delete(.:format)     rails_admin/main#bulk_delete
bulk_action POST        /:model_name/bulk_action(.:format)     rails_admin/main#bulk_action
       show GET         /:model_name/:id(.:format)             rails_admin/main#show
       edit GET|PUT     /:model_name/:id/edit(.:format)        rails_admin/main#edit
     delete GET|DELETE  /:model_name/:id/delete(.:format)      rails_admin/main#delete
show_in_app GET         /:model_name/:id/show_in_app(.:format) rails_admin/main#show_in_app
```

In this case its 'user' but it happens with all models. I've spent some time to dig into this but couldn't find anything.  It works fine with rails 4.2.6 version. Any ideas?

**Comment by YoranBrondsema** [comment]
:+1:

**Comment by mshibuya** [comment]
Fixed by cebaa0ea76f6a9085cc539af154f9695393caac9.
